docs/usage/self-hosted-configuration.md

```diff
@@ -149,8 +149,11 @@ Supported tools for dynamic install are:
 - `jb`
 - `jsonnet-bundler`
 - `mix`
-- `npm`
+- `npm` (including `yarn`, `pnpm` and `lerna`)
```

Comment:
```suggestion
- `npm`
- `yarn`
- `pnpm`
- `lerna`
```

How about this? This way the list always has one item per line?

Reply:
i think we should add also node and python then

Reply:
Sounds good to me. I'll let you add those to the list.

You'll probably need to sort the list from A-Z after adding the new items. 🙂